how-to/workspace-platform-starter/client/types/module/shapes/app-shapes.d.ts

```diff
@@ -104,10 +104,45 @@ export interface LaunchPreference {
  * Additional options that apply to a view
  */
 export interface ViewLaunchPreference {
+	/**
+	 * If specified it indicates wish to specify specific host settings for this content.
+	 */
+	host?: HostLaunchPreference;
+}
+/**
+ * Additional options that apply to the host of the content
+ */
+export interface HostLaunchPreference {
 	/**
 	 * If specified it indicates you do not want to use a browser window for this view but a platform window.
 	 */
-	hostUrl?: string;
+	url?: string | undefined;
```

Comment:
All these types are already optional, they shouldn't need `| undefined`

Reply:
It doesn't this was fixed in the source but needed to regenerate types and missed that step. Good catch.